google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -87,12 +89,18 @@ appengine.flex.config.project.missing.message=Please select a project.
 appengine.flex.config.user.specified.artifact.title=Select a JAR or WAR file
 appengine.flex.config.user.specified.artifact.error=Browse to a JAR or WAR file.
 appengine.flex.config.custom.deploy.title=App Engine flexible environment Configuration
-appengine.flex.config.custom.app.yaml.error=Browse to an app.yaml file.
-appengine.flex.config.custom.dockerfile.error=Browse to a Dockerfile.
+appengine.flex.config.browse.app.yaml=Browse to a YAML file.
+appengine.flex.config.browse.dockerfile=Browse to a Dockerfile.
+appengine.flex.config.badchars=The specified {0} path contains invalid characters or starts or ends with spaces.
 appengine.config.deployment.source.error=Select a valid deployment source.
 appengine.config.version.error=Enter a version ID.
-appengine.flex.deployment.cost.warning={0}There is no free quota for App Engine flexible environment deployments. Please visit {1}GCP Pricing{2} for pricing information.{3}
+appengine.flex.deployment.name=Google App Engine Flexible Deployment
+appengine.flex.deployment.cost.warning.beforeLink=There is no free quota for App Engine flexible environment deployments. Visit 
+appengine.flex.deployment.cost.warning.link=GCP Pricing
+appengine.flex.deployment.cost.warning.afterLink=for pricing information
 appengine.flex.user.specified.deploymentsource.name=Filesystem JAR or WAR file
+appengine.flex.facet.files.inexisting=Some configuration files do not exist.
+appengine.flex.facet.nodockerfile=Dockerfile can't be specified because runtime is set to "java" in the YAML configuration file.
```

Comment:
This is for when you specify a YAML file with "runtime: java". In the facet screen, we're actually disabling rather than hiding the Dockerfile text box, and adding this label underneath to "educate" users.